manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -435,12 +435,12 @@ data:
               {{- end}}
             ]
         - name: ISTIO_META_CLUSTER_ID
-          value: "{{ valueOrDefault .Values.global.multiCluster.clusterName `Kubernetes` }}"
+          value: "{{ valueOrDefault .Cluster `Kubernetes` }}"
         - name: ISTIO_META_INTERCEPTION_MODE
           value: "{{ or (index .ObjectMeta.Annotations `sidecar.istio.io/interceptionMode`) .ProxyConfig.InterceptionMode.String }}"
-        {{- if .Values.global.network }}
+        {{- if .Network }}
```

Comment:
I'm not sure changing strings is safe - at least should require tools, release notes, etc. 

Can you just keep the old ones - or if adding new ( because the old ones are bad ? ) support both for at least a release ? 

But I think we're not at a point to bike-shed on names in values - if there is one in ProxyConfig we should use it, otherwise the discussion on how to name it should be in the api repo. Until this happens - keep old names.

Reply:
+1 let's change these in a follow-on effort.

Reply:
I've made it so I just edit the values map instead of adding new values for the template. No template changes should be required now.

Reply:
I don't fully understand why this is unsafe? Is this template used elsewhere? These aren't new API names, I'm adding Go template values to let us select between the network/cluster in config vs labels. The `Cluster` is more specific to centralistio code cleanup.